lib/findingaids/ead/component.rb

```diff
@@ -65,6 +65,7 @@ def to_solr(solr_doc = Hash.new)
     Solrizer.insert_field(solr_doc, "dao",        get_ead_dao_facet,                            :facetable)
     Solrizer.insert_field(solr_doc, "place",      get_ead_places,                               :displayable, :facetable)
     Solrizer.insert_field(solr_doc, "subject",    get_ead_subject_facets,                       :facetable)
+    Solrizer.insert_field(solr_doc, "breadcrumb", bread_crumb(solr_doc),                        :displayable)
```

Comment:
@eshadatta You can also delete this and the above helper if you follow my previous suggestions